### PR TITLE
Add the slides of CNTUG meetup #8

### PR DESCRIPTION
**Pull request by hwchiu**
待相關投影片都收集完畢後會在修改標題

### CODE REVIEW

**Comment by kairen** [comment]
LGTM

File: communities/CNTUG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
         - [2018/07/14](#20180714)
         - [2018/07/30](#20180730)
         - [2018/08/11](#20180811)
+        - [2018/09/10](#20180908)
```
Comment (sufuf3) [inline communities/CNTUG.md]: Need to modify to `[2018/09/08](#20180908)`?
Thanks

File: communities/CNTUG.md [hunk 2]
```diff
@@ -225,7 +226,31 @@ CNCF 官方認證 Meetup: https://www.meetup.com/Cloud-Native-User-Group-Taiwan/
         - Slides: https://www.slideshare.net/hongweiqiu/overview-of-kubernetes-network-functions
     - Vikube - a kubernetes operation tool in Vim
         - Speaker: 林佑安(C9S)
-        - Slides: 
+        - Slides: TBA
```
Comment (sufuf3) [inline communities/CNTUG.md]: Let's add https://speakerdeck.com/c9s/vikube-operate-kubernetes-in-vim
Thanks

**Comment by hwchiu** [comment]
@sufuf3  Thanks your review and all fix.

**Pull request merged**